tests/text_search.rs

```diff
@@ -43,14 +43,12 @@ async fn test_text_search() {
                 assert!(place.icon.is_some());
                 assert!(place.icon_background_color.is_some());
                 assert!(place.icon_mask_base_uri.is_some());
-                assert!(place.opening_hours.is_some());
                 assert!(place.photos.as_ref().map(|vec| vec.len()).unwrap_or(0) > 0);
                 assert!(place.plus_code.is_some());
                 assert!(place.types.is_some());
                 // assert!(place.vicinity.is_some()); // this is occasionally null
```

Comment:
does vicinity exist in PlaceSearchPlace struct? Maybe give these comments one more pass-through and then we should be able to merge.

Check this for Tests/Examples in TextSearch, NearbySearch, and FindPlace.

Reply:
https://developers.google.com/maps/documentation/places/web-service/place-data-fields

All tests are fixed. Referred to api doc listed above to double check what fields are supported in Find Place, Nearby Search and Text Search.